go/gilded_rose.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import "fmt"
+
+type GildedRose struct {
+	Items []Item
+}
+
+type Item struct {
+	Name            string
+	SellIn, Quality int
+}
+
+func main() {
+	rose := GildedRose{
```

Comment:
Make GildedRose not testable is a (original kata) choice: you're first step must be to make code testable.

If you do make code already testable, you lost a part of the kata.